app/models/internal_post.rb

```diff
@@ -0,0 +1,29 @@
+class InternalPost < Post
+  belongs_to :user
+  validates :user, presence: true
+  has_many :comments, as: :parent
+
+	before_save do
+    parser = Redcarpet::Markdown.new(Redcarpet::Render::HTML, autolink: true)
+    self.body_html = parser.render(body_markdown)
+  end
+
+  after_create :notify_twitter
+  # TODO: move this to background job
+
+  def notify_twitter
+    $twitter_client.update(tweet_content)
+  end
+
+  def tweet_content
+    url = Rails.application.routes.url_helpers.
+      post_short_link_url(self, host: 'rbga.me')
```

Comment:
Align the operands of an expression in an assignment spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.